modules/images/dominant-color/class-wp-image-editor-gd-with-color.php

```diff
@@ -0,0 +1,69 @@
+<?php
+/**
+ * WordPress Image Editor Class for Image Manipulation through GD
+ * with dominant color detection
+ *
+ * @package performance-lab
+ * @group dominant-color
+ *
+ * @since 1.0.0
+ */
+
+/**
+ * WordPress Image Editor Class for Image Manipulation through GD
+ * with dominant color detection
+ *
+ * @since 6.x
+ *
+ * @see WP_Image_Editor
+ */
+class WP_Image_Editor_GD_With_Color extends WP_Image_Editor_GD {
+
+	/**
+	 * Get dominant color from a file.
+	 *
+	 * @param string $default_color default is light grey.
+	 *
+	 * @return string
```

Comment:
Let's add `@since` tags throughout. Also, please make sure to provide clear documentation, e.g. particularly here there's a caveat that needs to be clarified in docs, that there should not be a leading `#`.

```suggestion
	 * @since n.e.x.t
	 *
	 * @param string $default_color Optional. Hex color string, without leading #. Default is light grey.
	 * @return string Dominant hex color string.
```